WooCommerce/Classes/ViewRelated/Dashboard/Onboarding/StoreOnboardingView.swift

```diff
@@ -82,17 +97,20 @@ struct StoreOnboardingView: View {
                 StoreSetupProgressView(isExpanded: viewModel.isExpanded,
                                        totalNumberOfTasks: viewModel.taskViewModels.count,
                                        numberOfTasksCompleted: viewModel.numberOfTasksCompleted,
-                                       shareFeedbackAction: shareFeedbackAction)
+                                       shareFeedbackAction: shareFeedbackAction,
+                                       isRedacted: viewModel.isRedacted)
 
                 // Task list
                 VStack(alignment: .leading, spacing: Layout.verticalSpacingBetweenTasks) {
                     ForEach(viewModel.tasksForDisplay) { taskViewModel in
                         let isLastTask = taskViewModel == viewModel.tasksForDisplay.last
 
                         StoreOnboardingTaskView(viewModel: taskViewModel,
-                                                showDivider: !isLastTask) { task in
+                                                showDivider: !isLastTask,
+                                                isRedacted: viewModel.isRedacted) { task in
                             taskTapped(task)
                         }
+                                                .shimmering(active: viewModel.isRedacted)
```

Comment:
I learnt about the redacted and shimmering from your Domain and Store creation-related code. 🙇 

I am sending the redacted state to `StoreOnboardingTaskView` because I am showing/hiding the chevron image based on the redacted state [here](https://github.com/woocommerce/woocommerce-ios/pull/9033/files#diff-b0ddaa424d3882038d9f4a7c1007f3f7c4a8cbe6fd1917ce2bb2600474f7a717R60)

Please correct me if I am missing something.